### PR TITLE
compatible to namespaced phpunit classes

### PR DESCRIPTION
**Pull request by DQNEO**
As of PHPUnit 6, Long name classes (`\PHPUnit_Framework_*`)  don't exist any more.
So we should prepare for it.

### CODE REVIEW

**Comment by DQNEO** [comment]
Sure! did it. 1e6b06a9e5e8dfcf993d800decb84d810b821634

File: .changes/3.64.16 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[
```
Comment (kstich) [inline .changes/3.64.16]: Missed this in the other PR so I'm submitting an update, but these should go in a unique file in the `.changes/nextrelease/` folder.

**Comment by DQNEO** [comment]
OK, both PRs are made for the same purpose, I merged the messages into one line.

**Comment by kstich** [comment]
Still needs to move to the proper location like `.changes/nextrelease/namespaced_phpunit_classes.json`.

**Comment by DQNEO** [comment]
Oh, sorry I completely misunderstood the rule.
I renamed it.  73d72101

**Pull request merged**